esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -254,6 +254,16 @@ diagnostics:
          type: sat, version: NOAA, start_year: 1982, end_year: 2016}
     scripts: null
 
+  PIOMAS:
+    description: PIOMAS
+    variables:
+      sithick:
+        mip: day
+        fx_files: [areacello]
```

Comment:
Note that this syntax is no longer supported since https://github.com/ESMValGroup/ESMValCore/pull/170 was merged.